client/extensions/woocommerce/woocommerce-services/views/packages/package-dialog.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { concat, difference, flatten, map } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import EditPackage from './edit-package';
+import checkInputs from './modal-errors';
+import Dialog from 'components/dialog';
+import FormSectionHeading from 'components/forms/form-section-heading';
+import FormButton from 'components/forms/form-button';
+import inputFilters from './input-filters';
+import PredefinedPackages from './predefined-packages';
+import SegmentedControl from 'components/segmented-control';
+
+const getDialogButtons = ( mode, dismissModal, savePackage, onRemove, translate ) => {
+	const buttons = [
+		<FormButton onClick={ savePackage }>
+			{ ( 'add-custom' === mode ) ? translate( 'Add package' ) : translate( 'Apply changes' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **56** times:
`translate( 'Apply Changes' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/16610-packages-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).